test/test_vmap.py

```diff
@@ -2986,26 +2985,21 @@ class TestVmapOperatorsOpInfo(TestCase):
         xfail('tensor_split'),
         xfail('to_sparse'),
         xfail('vsplit'),
-        xfail('hstack'),
-        xfail('vstack'),
-        xfail('dstack'),
         xfail('linalg.multi_dot'),
         xfail('nanmean'),
-        xfail('block_diag'),
-        xfail('nn.functional.dropout'),
 
-        # entries in here need don't work and need to be fixed.
+        # entries in here don't work and need to be fixed.
         # Each one of these is a bug
-        xfail('unfold'),
+        # xfail('unfold'),
         xfail('svd', device_type='cuda'),
         xfail('linalg.svd', device_type='cuda'),
-        xfail('index_put'),
-        xfail('nn.functional.max_pool2d'),
-        xfail('nn.functional.batch_norm'),
+        # xfail('nn.functional.max_pool2d'),
+        # xfail('nn.functional.batch_norm'),
```

Comment:
Commented entries for which tests failed as "unexpected failure". Not sure if this works. cc @Chillee